sectionsUtilityTool/src/main/webapp/assets/js/utils.js

```diff
@@ -187,7 +187,7 @@ $(document).on('click', '.postCourseNameChange', function (e) {
   e.preventDefault();
   var thisCourse = $(this).attr('data-courseid');
   var newCourseName = $(this).closest('.courseTitleTextContainer').find('input.courseTitleText').val();
-  var url = 'manager/api/v1/courses/' + thisCourse + '?course[course_code]=' + newCourseName + '&course[name]=' + newCourseName;;
+  var url = 'manager/api/v1/courses/' + thisCourse + '?course[course_code]=' + newCourseName + '&course[name]=' + newCourseName;
```

Comment:
Is a direct call to the API secure?
If the prefix part of the API call is also used in the servlet then it should probably be supplied to the UI rather than hard coded.  This would allow changing the value of the API prefix everywhere via changing a property rather having to hunt down occurrences in code.